chainer/links/connection/mlp_convolution_2d.py

```diff
@@ -44,8 +44,7 @@ class MLPConvolution2D(link.ChainList):
     """
 
     def __init__(self, in_channels, out_channels, ksize, stride=1, pad=0,
-                 activation=relu.relu, wscale=1,
-                 conv_init=None, bias_init=None):
+                 activation=relu.relu, conv_init=None, bias_init=None):
```

Comment:
We need to make `conv_init` and `bias_init` keyword arguments and prohibit to make them positional ones. I think it is an acceptable compromisation.